web/server/web_client.c

```diff
@@ -1490,7 +1490,11 @@ static inline int web_client_process_url(RRDHOST *host, struct web_client *w, ch
     strncpyz(filename, w->last_url, FILENAME_MAX);
     tok = mystrsep(&url, "?");
     buffer_flush(w->response.data);
-    return mysendfile(w, (tok && *tok)?tok:"/");
+    char *file_selection = (tok && *tok)?tok:"/";
+    if(web_client_default_dashboard == WEB_SERVER_REACT_DASHBOARD && !strcmp(file_selection, "/"))
```

Comment:
Does this comparison with "/" work for all cases?

Reply:
Yes, I could not see any problems with my tests.

Reply:
No mention of React or any specific version of the dashboard (or other javascript code) should be in the C code.